src/Fixer/Phpdoc/PhpdocToCommentFixer.php

```diff
@@ -88,6 +88,10 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
+            if ($commentsAnalyzer->isAnnotation($token)) {
```

Comment:
Can we have this configurable to maintain BC? An option `ignore_annotations` (with default to `false`) seems the way.